### DISCUSSION TITLE
Welcome to yandex_music_fisher_mod Discussions!

### DISCUSSION THREAD

**Discussion by vectorserver** [vectorserver/yandex_music_fisher_mod#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by d1saw** [comment]
привет, при нажатии на кнопку "авторизация", выдает следующее:
![image](https://github.com/user-attachments/assets/b45773f7-0976-4e96-b1d6-1515403fb35c)

**Reply by vectorserver** [reply]
Ага, спасибо поправил!

**Comment by BusyBo55** [comment]
Привет, при авторизации вот что пишет
![Снимок экрана 2025-06-15 134039](https://github.com/user-attachments/assets/172178e9-4286-40ad-897c-9e96b2d1c1f2)

Фото консоли
![Снимок экрана 2025-06-15 134706](https://github.com/user-attachments/assets/903c8872-5397-45d9-8bc7-bb1cd5b723d1)

Сами скрипты в развороте
![image](https://github.com/user-attachments/assets/f7c55ef4-1aef-4d50-a664-90dd59dde20b)
и так далее. Если нужно - досниму ниже

**Comment by OnePunMan1337** [comment]
Здравствуйте! Планируется ли возможность загрузки треков в наивысшем качестве (flac, если же конечно, он доступен у трека) ?

**Comment by Ayrik161RUS** [comment]
не работает нельзя скачать песни. 
<img width="1174" height="550" alt="Снимок экрана 2026-03-20 225710" src="https://github.com/user-attachments/assets/4c4a16d0-d604-4383-91ae-96ee778b21e2" />

**Comment by Richigo15** [comment]
Доброго времени суток. Планируется ли функция скачивания трека вместе с текстом песни, если он присутствует? Или я где-то упустил этот функционал в настройках?
